src/js/components/Accordion/__tests__/__snapshots__/Accordion-test.js.snap

```diff
@@ -220,9 +220,9 @@ exports[`Accordion AccordionPanel 1`] = `
 
 @media only screen and (max-width:768px) {
   .c6 {
-    font-size: 18px;
-    line-height: 24px;
-    max-width: 432px;
+    font-size: 16px;
+    line-height: 22px;
+    max-width: 384px;
```

Comment:
Can you add a few words from your analysis why the snapshots had changed and whether that's a backward compatibility concern?

Reply:
This is a heading level 4 that resized to a level 5. Before this pr heading level 4 never resized. I don't see this as a backwards compatibility issue, as the heading not resizing above level 4 was a bug.

Reply:
I agree, thanks for the inputs.